velox/duckdb/README.md

```diff
@@ -8,13 +8,12 @@ a new feature, first clone DuckDB git repository:
 
 Then generate the amalgamated .cpp and .hpp files:
 
-    python scripts/amalgamation.py --extended
+    python scripts/amalgamation.py --extended --splits=8
```

Comment:
I just landed a PR earlier today to fix the instructions after splitting the amalgamated files. I guess you might see a conflict if you rebase it.